### PR TITLE
Task id thread suffix

### PR DESCRIPTION
**Pull request by midnightercz**

### CODE REVIEW

**Review by lrossi-dev — APPROVED** [review]

**Comment by emilyzheng** [comment]
Could you remove the AWS settings? Otherwise, it looks good.

File: pubtools/_quay/tag_docker.py [hunk 1]
```diff
@@ -743,6 +743,7 @@ def untag_image(self, push_item: Any, tag: str) -> None:
 
         cert, key = get_pyxis_ssl_paths(self.target_settings)
 
+        set_aws_kms_environment_variables(self.target_settings, "cosign_signer")
```
Comment (emilyzheng) [inline pubtools/_quay/tag_docker.py:746]: Remove this as well?

Comment (midnightercz) [reply pubtools/_quay/tag_docker.py]: done

File: pubtools/_quay/tag_docker.py [hunk 2]
```diff
@@ -794,6 +794,7 @@ def manifest_list_remove_archs(self, push_item: Any, tag: str, remove_archs: lis
                 (to_remove_man["digest"], tag, list(push_item.repos.keys())[0])
             )
 
+        set_aws_kms_environment_variables(self.target_settings, "cosign_signer")
```
Comment (emilyzheng) [inline pubtools/_quay/tag_docker.py:797]: And here

**Review by emilyzheng — APPROVED** [review]

**Pull request merged**